samples/tls/logging/README.md

```diff
@@ -2,22 +2,9 @@
 
 ## Overview
 
-This sample demonstrates how to deploy logging components with TLS enabled.
-For now, only ingress is covered. In-cluster TLS will be supported in the
-future.
+This sample demonstrates how to deploy logging components with TLS enabled for connections to Kibana.
```

Comment:
It also shows TLS ingress for Elasticsearch, although I'm not sure why as we would generally like to keep Elasticsearch isolated in-cluster for security reasons. Greg?

Reply:
Some sites may choose to make Elasticsearch visible to their users.  This would enable, for example, users to submit Elasticsearch queries via API calls or scripts using one of the query languages.  We may need/want to add a sentence or two explaining that enabling TLS for Kibana and enabling TLS for Elasticsearch are independent decision.  Hmmm, that probably ties into a higher-level discussion of whether they want to make Elasticsearch directly available or not.  We make the es_nodeport_enable.sh script available to allow nodePort access to ES since we do not enable that by default.  I wonder if that needs its own README or, probably better, adding a paragraph in the top-level Logging README file.

Reply:
Section added to main logging README